recipes/cryptopp/all/CMakeLists.original.txt

```diff
@@ -0,0 +1,1240 @@
+# Please ensure your changes or patch meets minimum requirements.
```

Comment:
Where did you obtain it?

Reply:
bincrafters

Reply:
I think it could be simplified a lot, but as it's an external project. I would prefer downloading the files directly instead of copying here.

You could add tools.download to the recipe.
The link to the cmake file: https://github.com/noloader/cryptopp-cmake/blob/master/CMakeLists.txt
Of course, you should use a specific commit (latest) to avoid any possible unstable behavior.

Reply:
Why is this using a custom CMakelists.txt instead of the original build files of the library? Also naming it with "original" is a bit misleading I think